hax/treemakers/common.py

```diff
@@ -162,6 +162,27 @@ def extract_data(self, event):
                     result['largest_other_s2_pattern_fit'] = getattr(rp, 'goodness_of_fit')
                     break
 
+        # get information for MisIdS1SingleScatter cut
+        result['largest_s2_before_main_s2_area'] = float('nan')
+        result['largest_s2_before_main_s2_time'] = float('nan')
+        result['secondlargest_s2_before_main_s2_area'] = float('nan')
+        result['secondlargest_s2_before_main_s2_time'] = float('nan')
+
+        # grab the two largest s2s that occurs in time before the main S2
+        suspicious_s2s = []
+        for s2_i in event.s2s:
+            if event.peaks[s2_i].hit_time_mean < s2.hit_time_mean:
+                suspicious_s2s.append(event.peaks[s2_i])
+            if len(suspicious_s2s) >= 2:
+                break
+
+        for s2,key1, key2 in zip(suspicious_s2s, ['largest_s2_before_main_s2_area',
+                                                  'secondlargest_s2_before_main_s2_area'],
+                                 ['largest_s2_before_main_s2_time',
+                                  'secondlargests2_before_main_s2_time'
+                              ]):
```

Comment:
__[Flake8]__

[E124] closing bracket does not match visual indentation


<!-- Comment by [Sider](https://sider.review) -->